### ISSUE TITLE
Change default site title and favicon

### ISSUE THREAD

**Issue by dig9074vijay** [DhairyaBahl/React-Messenger-App#48]
Hey @DhairyaBahl !
I just noticed that we are still using the default the site title and favicon provided with the React boilerplate code.
I think we should change it to something matching our site's objective. [GSSOC`21 participant]
<img width="203" alt="Screenshot 2021-03-10 at 11 16 22 PM" src="https://user-images.githubusercontent.com/55308273/110673313-a1b71400-81f6-11eb-8f48-e3fe9b573dd2.png">

**Comment by DhairyaBahl** [comment]
@dig9074vijay You can change the favicon because title is already fixed ! Thanks :)

**Comment by dig9074vijay** [comment]
@DhairyaBahl I just went through the issues . This issue existed before and has been fixed already.

**Comment by DhairyaBahl** [comment]
@dig9074vijay You are absolutely right ! and I just deployed all the changes to the website but still favicon is not changed. You can check it out that  what exactly the error is ! and why favicon is not showing up !

**Comment by dig9074vijay** [comment]
Yeah! Thanks @DhairyaBahl I would like to look into it.😁

**Comment by DhairyaBahl** [comment]
@dig9074vijay Then I will assign you this issue.
